### DISCUSSION TITLE
@formily/antd 的 ArrayTable 对树形结构的兼容问题

### DISCUSSION THREAD

**Discussion by wwyx778** [alibaba/formily#3270] [Q&A 2.x] [answered]
CodeSandbox: https://codesandbox.io/s/formily-tree-table-0v0lq8?file=/App.tsx

发现 ArrayTable 对树形结构的支持还不是很好，目前有以下几个问题:

 - 子行的排序 `Index` 展示没有做处理，容易和父行混淆
 - 子行的拖拽排序不可用❌（样式问题，key重复问题）
 - 子行设初始值无效，所设初始值和编辑表单提交的表单值是一致的

**Comment by janryWang** [comment, marked as answer]
一直就没考虑过支持Tree，这里要做的事情很多的

**Reply by wwyx778** [reply]
:-( 未来有支持计划吗？

**Reply by janryWang** [reply]
没有，自己封装吧
